packages/parchment/lib/src/document/embeds.dart

```diff
@@ -98,6 +98,9 @@ class BlockEmbed extends EmbeddableObject {
   }) : super(inline: false);
 
   static final BlockEmbed horizontalRule = BlockEmbed('hr');
-  static BlockEmbed image(String source) =>
-      BlockEmbed('image', data: {'source': source});
+  static BlockEmbed image(String source, Map<String, dynamic> data) {
```

Comment:
This is introducing a breaking change in the API, can `data` be an optional parameter?

Reply:
@marcio-ota  It doesn't seem you have addresses this comment